### ISSUE TITLE
Tests failing due to vispy deprecation 

### ISSUE THREAD

**Issue by tlambert03** [napari/napari#3693]
## 🐛 Bug
Tests are now failing because we use a vispy API internally that we also deprecated 😄 

```
          if scaling is not None:
  >           warnings.warn(
                  "The scaling parameter is deprecated. Use MarkersVisual.scaling instead",
                  DeprecationWarning,
                  stacklevel=2,
              )
  E           DeprecationWarning: The scaling parameter is deprecated. Use MarkersVisual.scaling instead
```

didn't look to deeply, but it wasn't immediately clear to me what we need to change to remove the warning.  @brisvag, can you update our code for this?

**Comment by brisvag** [comment]
These deprecations are fixed by #3494; however, that one is blocked due to another CI failure, which in turn is seemingly unrelated and popping up in other PRs (macos tests).

**Comment by Carreau** [comment]
I'm still tracking down the core of those mac failures. It's still unclear. I have found many related issues that I'm fixing one by one. Maybe we should skip those test for now on mac ?

**Comment by Carreau** [comment]
(I'm hopping #3695 might help??)

**Comment by sofroniewn** [comment]
@Carreau is it ok if i go ahead and merge in #3494 now? and then we update tests in a followup PR

**Comment by Carreau** [comment]
> @Carreau is it ok if i go ahead and merge in #3494 now? and then we update tests in a followup PR

Sure !

**Comment by brisvag** [comment]
Closed by #3494.
